dev/NavigationView/NavigationView.cpp

```diff
@@ -4515,6 +4525,15 @@ void NavigationView::UpdateTitleBarPadding()
     }
 }
 
+void NavigationView::OnAutoSuggestBoxQuerySubmitted(const winrt::AutoSuggestBox& sender, const winrt::Windows::UI::Xaml::Controls::AutoSuggestBoxQuerySubmittedEventArgs& args)
```

Comment:
would we get this event if there were no matches for the query ? If there is no selection in that case, perhaps we should keep the pane open.

Reply:
That's a very good point. I think the SuggestionChosen event is more appropriate in this case, do you agree?

Reply:
That might be a better event. There is also an args on this event that has some suggestions, but I don't think that guarantees a selection.